benchmark/python/output

```diff
@@ -0,0 +1,13 @@
+Batch Size,1.0
```

Comment:
do we need to commit the output file into repo?

Reply:
Yes, I am about to do that. Thanks!

Reply:
any update on this? we can merge after removing the output